backends/tpp/playbook.md

```diff
@@ -0,0 +1,19 @@
+# TPP Operations 
+
+See the [README](./README.md) for getting started information.
+
+## Removing medium privacy outputs
+All outputs are put into the high privacy directory of a workspace on L3. Outputs that have been marked as having a lower privacy level are then copied into the appropriate directory (i.e at the moment only the medium privacy directory).
+
+Medium privacy outputs are copied from L3 to L4 every five minutes by a sync script controlled by TPP. *Note: this sync script is unidirectional, so any changes to L4 are not reflected back on L3.*
+
+There are times when these medium privacy outputs may need to be deleted. For example, the researcher or output checkers may realise they should have been marked as high privacy, or the researcher may no longer need the output and want to preserve disk space.
+
+To remove a medium privacy output:
+1. On L3, as the job runner user
```

Comment:
typo - obviously a user would understand what's intended, but we are planning to rename the user used to manage things like this so better to ensure it will easily match a grep